app/assets/javascripts/send_preview.js

```diff
@@ -1,3 +1,4 @@
+var exports;
```

Comment:
Same here, what does this change have to do with this PR 🤔 ?

Reply:
JShint again...

Reply:
Same as what I said for the other JS file - this is for sure already fixed on master.